### DISCUSSION TITLE
New "Add new style..." behavior: not inheriting the format

### DISCUSSION THREAD

**Discussion by euu2021** [freeplane/freeplane#656] [Ideas]
Shouldn't the new style inherit all the formatting from the selected node? In this example, the node has a a lot of changes in the formatting, but the new style inherited nothing:

![javaw_7OXHm1EkOa](https://user-images.githubusercontent.com/77707706/189506678-c73ee212-6fb6-41f1-a5e1-c37263b94832.gif)

Example used:
[EXAMPLE new style.zip](https://github.com/freeplane/freeplane/files/9541603/EXAMPLE.new.style.zip)

**Comment by dpolivaev** [comment]
I changed it in the last preview as you suggest, and I would like to ask for feedback from the community.
Version 1.10.5_02
